services/users.services.js

```diff
@@ -10,10 +10,10 @@ async function login({ username, password }, callback) {
       const token = auth.generateAccessToken(username);
       return callback(null, { ...user.toJSON(), token });
     } else {
-      return callback({ message: "Invalid USername/PassWord" });
+      return callback({ message: "Invalid Username/Password" });
```

Comment:
the error msg should also be in string file

Reply:
done